paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -440,30 +440,6 @@ internal class PaymentSheetActivityTest {
         }
     }
 
-    @Test
-    fun `updates navigation button`() {
```

Comment:
This is tested with the new Compose and screenshot tests. The `bottomSheetBehavior` that’s included in this test is also tested elsewhere.